lib/internal/fs/sync.js

```diff
@@ -53,6 +56,25 @@ function copyFile(src, dest, mode) {
   );
 }
 
+function mkdtemp(prefix, options) {
+  options = getOptions(options);
```

Comment:
hmm, not sure since we we started to move the sync implementations to a different file, but I gotta say I am not a fan because 1) this incurs an unnecessary hit to startup (even with deserialization) and 2) moving code around makes `git blame` less useful, especially when it's useful to have the sync and the async version side by side for cross-reference..